### PR TITLE
etcd_worker: fix a bug that etcd worker maybe lost data at starting

### PR DESCRIPTION
**Pull request by leoppro**
### What problem does this PR solve? 

 - Fix a bug that etcd worker lost data
 - a etcd worker bank test

### What is changed and how it works?
the etcd worker have two steps when it's starting:
1. read entire data snapshot and apply it to the reactor state
2. start to watch the data change log from etcd and apply logs to the reacator state

the data lost between step 1 and step 2 because the start reversion of the watch is not set

### Check List 

Tests 

 - Unit test

### Release note

- No release note

### CODE REVIEW

File: pkg/orchestrator/etcd_worker_bank_test.go [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+	bankState.notFirstTick = true
+	return state, err
+}
+
+func (s *etcdWorkerSuite) TestEtcdBank(c *check.C) {
+	defer testleak.AfterTest(c)()
+	totalAccountNumber := 25
+	workerNumber := 10
+	var wg sync.WaitGroup
+	// ctx, cancel := context.WithTimeout(context.Background(), time.Minute*5)
```
Comment (leoppro) [inline pkg/orchestrator/etcd_worker_bank_test.go:126]:
```suggestion
```

**Comment by ben1009** [comment]
/ l g t m

**Review by amyangfei — APPROVED** [review]

**Review by lichunzhu — COMMENTED** [review]
/lgtm

**Pull request merged**